h/static/scripts/annotator/guest.coffee

```diff
@@ -211,21 +161,131 @@ module.exports = class Guest extends Annotator
 
     this.removeEvents()
 
-  setupAnnotation: ->
-    annotation = super
-    this.plugins.CrossFrame.sync([annotation])
-    annotation
+  setupAnnotation: (annotation) ->
+    self = this
+    root = @element[0]
+
+    anchors = []
```

Comment:
A brief comment here explaining what these three containers are for ("what is a dead highlight? what is the difference between an anchor and an anchoredTarget?") would be good.